packages/react-core/src/components/Card/examples/Card.md

```diff
@@ -98,23 +107,57 @@ A common use case of this is to set all but one body section to `isFilled={false
 
 ```
 
+### Clickable cards
+
+A card can perform an action or navigate to an external link by clicking anywhere within the card.
+
+When a card is meant to be clickable only, you must avoid rendering any other interactive content within the `<Card>`.
+
+```ts file='./CardClickable.tsx'
+
+```
+
 ### Selectable cards
 
-Selectable cards can only be selected one at a time, and are intended for use with [primary-detail layout](/demos/primary-detail).
+A card can also be selected by clicking anywhere within the card.
+
+Similar to clickable only cards, you must avoid rendering any other interactive content within the `<Card>` when it is meant to be selectable only. Refer to our [clickable and selectable example](#clickable-and-selectable-cards) if you need a card that is both selectable and has other interactive content.
 
 ```ts file='./CardSelectable.tsx'
 
 ```
 
+### Single selectable cards
+
+When a group of single selectable cards are related, you should pass the same `name` property to each card's `selectableActions` property.
+
+```ts file='./CardSingleSelectable.tsx'
+
+```
+
+### Clickable and selectable cards
+
+A card can be selectable and have additional interactive content by passing both the `isClickable` and `isSelectable` properties to `<Card>`. The following example shows how the "clickable" functionality can be rendered anywhere within a selectable card.
+
+When passing interactive content to a clickable and selectable card that is disabled, you should also ensure the interactive content is disabled as well, if applicable.
+
+```ts file='./CardClickableSelectable.tsx'
+
+```
+
 ### Legacy selectable cards
 
 The following example shows a legacy implementation of selectable cards. This example uses the `isSelectable` property instead of `isSelectableRaised`, which is the current recommendation for implementation. `isSelectable` applies selectable styling, but does not apply raised styling on hover and selection as `isSelectableRaised` does.
 
+A `tabIndex={0}` is also manually passed to allow the card to be focused and clicked via keyboard.
+
 ```ts file='./CardLegacySelectable.tsx'
 
 ```
 
-### Selectable card accessibility features
+### Legacy selectable card accessibility features
```

Comment:
I am wondering if we should move away from legacy here in title. We could call it deprecated like in the core examples or, I would even say maybe we should create a deprecated tab for theses examples.
@mcarrano what do you think?

Reply:
I agree that we should move away from the word "Legacy." I think calling them deprecated makes sense, but not sure we should create a separate tab.